aten/src/ATen/test/vec_test_all_types.cpp

```diff
@@ -1700,6 +1700,169 @@ namespace {
             << "Test failed for uint16 to float " << u16 << "\n";
       }
     }
+    TEST(FP8E4M3Test, FP8E4M3ConversionFloat) {
+      for (uint32_t index = 0; index < 256; ++index) {
+        uint8_t input_u8 = static_cast<uint8_t>(index);
+        float f32 = static_cast<float>(input_u8);
```

Comment:
I guess the purpose of iterating from 0 to 255 is to cover all the 8-bit values for FP8, not to cover 0.0 to 255.0 floating point values? Should the right approach to create FP8 by bitcasting `input_u8` to FP8 and then do conversions?

Reply:
oh, yes, sorry for the mistake. Will update it soon.

Reply:
Updated the UT @jgong5 please kindly help to take a look again.